ads_plugin.py

```diff
@@ -33,7 +35,7 @@
 import pandas as pd
 import imageio
 
-VERSION = "0.2.19"
+VERSION = "0.2.20"
```

Comment:
At line 54 below, we set the default value for overlap at 25. I would use the same value as elsewhere, which is 48.